frontend/spa/src/components/ImageLabeling.vue

```diff
@@ -2,134 +2,186 @@
-                    </form>
-                </div>
-                <div id="choose_overlap" class="col border-right">
-                    <span>Overlap Mode</span>
-                    <form>
-                        <input type="radio" class="radio-button" name="overlap_mode" value="overlap" v-model="active_overlap_mode" v-bind:class="{ disabled: isDisabled }" v-bind:disabled="stateOverlap"> Overlapping
-                        <br>
-                        <input type="radio" class="radio-button" name="overlap_mode" value="no-overlap" v-model="active_overlap_mode" v-bind:class="{ disabled: isDisabled }" v-bind:disabled="stateNoOverlap"> Non-overlapping
-                    </form>
-                </div>
-                <div id="semantic_labels" class="col border-right">
-                    <span>Label Classes</span>
-                    <form>
-                        <select v-model="active_label">                            
-                                <option v-for="label in labels" :key="label.label_class" name="semantic_label" :value="label.label_class">{{ label.label_class }}</option>
-                        </select>
-                    </form>
-                </div>
-                <div id="graphics_settings"  class="col border-right">
-                    <div id="stroke_slider_container">
-                        <span>Stroke thickness</span>
-                        <input id="stroke_thickness_slider" type="range" min="0" max="10" class="slider" v-model="stroke_slider_value">
-                    </div>
-                    <div id="opacity_slider_container">
-                        <span>Opacity</span>
-                        <input id="opacity_slider" type="range" min="0" max="100" class="slider" v-model="opacity_slider_value">
+                <div class="row justify-content-center">
+                    <div id="tools" class="col border-right">
+                        <div class="row">
+                            <div class="col">Drawing Tool
+                            </div>
+                        </div>
+                        <div class="row">
+                            <div class="col">
+                                <select width="100" style="width: 100px" id="tools_form" v-on:change="tool_selected" v-model="active_tool">
+                                    <option class="radio-button" name="tool" value="freehand"> Freehand </option>
+                                    <option class="radio-button" name="tool" value="polygon"> Polygon </option>
+                                    <option class="radio-button" name="tool" value="rectangle"> Rectangle </option>
+                                    <option class="radio-button" name="tool" value="point"> Point </option>
+                                    <option class="radio-button" name="tool" value="circle" Circle </option>
```

Comment:
option tag is missing a closing angle bracket - circle is currently not showing on drop down list